www/src/css/snaplist.css

```diff
@@ -15,6 +15,16 @@
   margin-right:10px;
 }
 
+.b-store .b-installer {
```

Comment:
It seems that `b-installer` is defined in different file (`installer.css`), so should be modified there.
And ideally it should be changed by adding modifier like `b-installer_flexible` (or any other name that makes sense in this context instead of nesting in `b-store`.

Same applies to `b-installer__button` below.

I understand that in many cases it's easier to have one class on parent component and have other styles depend on it, but if possible we should avoid such nesting (making styles dependent on parent element classes) and have proper state classes for every small component that needs it.